api/src/main/java/io/opencensus/trace/export/SampleStore.java

```diff
@@ -14,58 +14,51 @@
 package io.opencensus.trace.export;
 
 import static com.google.common.base.Preconditions.checkArgument;
-import static com.google.common.base.Preconditions.checkNotNull;
 
 import com.google.auto.value.AutoValue;
 import io.opencensus.trace.Span;
 import io.opencensus.trace.base.Status;
 import io.opencensus.trace.base.Status.CanonicalCode;
 import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
 import javax.annotation.concurrent.Immutable;
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * This class allows users to access in-process debugging information such as (getting access to all
- * active spans, support latency based sampled spans and error based sampled spans).
- *
- * <p>The active spans tracking is available for all the spans with the option {@link
- * Span.Options#RECORD_EVENTS}. This functionality allows users to debug stuck operations or long
- * living operations.
+ * This class allows users to access in-process information such as latency based sampled spans and
+ * error based sampled spans.
  *
  * <p>For all completed spans with the option {@link Span.Options#RECORD_EVENTS} the library can
  * store samples based on latency for succeeded operations or based on error code for failed
  * operations. To activate this, users MUST manually configure all the span names for which samples
  * will be collected (see {@link #registerSpanNamesForCollection(Collection)}).
  */
 @ThreadSafe
-public abstract class InProcessDebuggingHandler {
+public abstract class SampleStore {
 
-  InProcessDebuggingHandler() {}
+  protected SampleStore() {}
 
   /**
-   * Returns the summary of all available in-process debugging data such as number of active spans,
-   * number of sampled spans in the latency based samples or error based samples.
+   * Returns the number of sampled spans in all the latency buckets for every different span name.
    *
-   * <p>Latency based sampled summary buckets and error based sampled summary buckets are available
-   * only for span names registered using {@link #registerSpanNamesForCollection(Collection)}.
+   * <p>Data available only for span names registered using {@link
+   * #registerSpanNamesForCollection(Collection)}.
    *
-   * @return the summary of all available in-process debugging data.
+   * @return the number of sampled spans in all the latency buckets for every different span name.
    */
-  public abstract Summary getSummary();
+  public abstract Map<String, Map<LatencyBucketBoundaries, Integer>>
+      getNumberOfLatencySampledSpans();
```

Comment:
> public abstract Map<String, Map<LatencyBucketBoundaries, Integer>> getNumberOfLatencySampledSpans();

Would it be better to make a class for this return type?  Then you could change it to add more data in the future, if necessary.  The name would also be more descriptive.

Another improvement would be adding a `SpanName` class for the first key type, to improve readability and type safety.  This suggestion also applies to `ActiveSpans.getNumberOfActiveSpans()`.